fs2/src/main/scala/uk/gov/nationalarchives/dp/client/fs2/Fs2Client.scala

```diff
@@ -131,6 +132,9 @@ object Fs2Client:
       IO(createProcessMonitorClient(ClientConfig(url, secretName, LoggingWrapper(backend), duration, ssmEndpointUri)))
     }
 
+  // This bit might not be worth it. We have the other clients because their construction is quite complicated but this is pretty simple.
```

Comment:
This was just my comment to you :smile: Is it worth documenting which schemas can be validated against. You've got the XIP-V6 xsd now but we might add others.

Reply:
Hmm, it's generic enough that you could use any schema. Do we want to limit the schemas that could be used, to the ones that we have in `resources` folder of the Preservica client or do we want to also allow the ability to pass in a path to a schema on another computer/server? Would that even work?

Reply:
It would be good to use any path to any schema and not just the ones in the resources folder. In the `ValidateXmlAgainstXsd` you're calling `getClass.getResourceAsStream` which limits you somewhat. 
Would it be better to do something like 

```scala
val xipXsdSchemaV6 = getClass.getResourceAsStream("/XIP-V6.xsd") //Get this as a string

def apply[F[_]: Monad](schemaFileAsString: String): ValidateXmlAgainstXsd[F] =
    new ValidateXmlAgainstXsd[F](schemaFileAsString) {

      override def xmlStringIsValid(xmlStringToValidate: String): F[Boolean] =
```
Then you can pass in any schema and it should be fine. What do you reckon?